source/app/powertabeditor.cpp

```diff
@@ -2425,6 +2446,11 @@ void PowerTabEditor::createCommands()
                               tr("Slide Out Of Upwards"),
                               "TabSymbols.SlideOutOf.Upwards", QKeySequence(),
                               Note::SlideOutOfUpwards);
+    
+    myFingerCommand = new Command(tr("Finger..."), "TabSymbols.Finger",
+                                  QKeySequence(tr("Shift+F")), this);
```

Comment:
I don't think the key sequence should be a translatable string. I'd probably do `QKeySequence(Qt::SHIFT + Qt::Key_F)`